bundles/org.openhab.binding.teleinfo/src/test/resources/cbetm-base-option-1.raw

```diff
@@ -1,19 +1,19 @@
-MOTDETAT 000000 B
-
-ADCO XXXXXXXXXXXX 7
-OPTARIF BASE 0
-ISOUSC 20 8
-BASE 001181243 _
-PTEC TH.. $
-IINST1 000 H
-IINST2 002 K
-IINST3 000 J
-IMAX1 026 8
-IMAX2 018 :
-IMAX3 027 ;
-PMAX 07990 ?
-PAPP 00540 *
-MOTDETAT 400000 F
-PPOT 00 #
-
+MOTDETAT 000000 B

```

Comment:
Why is this file changed?

Reply:
Sorey don't know why.....how can I take it back?

Reply:
revert to origin